lib/internal/errors.js

```diff
@@ -164,6 +164,7 @@ E('ERR_SOCKET_BAD_PORT', 'Port should be > 0 and < 65536');
 E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
 E('ERR_V8BREAKITERATOR', 'full ICU data not installed. ' +
   'See https://github.com/nodejs/node/wiki/Intl');
+E('FALSY_VALUE_REJECTION', 'Promise was rejected with falsy value');
```

Comment:
dang... I should have caught this one before it landed... this needs the `ERR_` prefix on it. That should be fixed before this goes out in a release.

ping @refack @addaleax